source/energy/policy.rst

```diff
@@ -0,0 +1,37 @@
+.. policy:
+
+Modelling policies
+==================
+The global energy model distinguishes between eleven global regions (cf. Section :ref:`spatial`).  It is nevertheless important to represent current and planned national policies, such as the nationally determined contributions (NDCs) as agreed upon in Paris Agreement, at a lower geographical resolution in order to adequatly account for near term developments in scenario development processes. 
+
+Representation of single country Nationally Determined Contributions (NDCs)
+---------------------------------------------------------------------------
+The targets formulated in the NDCs come in many different flavors. This applies to the sectors and gasses, which policies cover, but also how these are expressed and quantified. In the global energy model, four broad categories of policy types related to the NDCs are represented, each of which is translated into a set of constraints.
+   1. Emission targets
+   2. Energy shares
+   3. Capacity or generation targets
+   4. Macro-economic targets
+A detailed description of the methodological implementation of the NDCs in the global energy model, along with an extensive list of the energy related targets considered can be found in Rogelj et al. (2017) :cite:`rogelj_indc_2017`. Additional policies implemented in the model can also be found in ('what reference for the CD_Links related policies?`)
```

Comment:
Please check CD_Links reference

Reply:
@volker-krey Marks paper is still not published. Do you have a suggestion for an alternative source to cite the currently implemented policies?

Reply:
@OFR-IIASA would this also work, or are you referring to other, later policy implementations?
Rogelj, J., Fricko, O., Meinshausen, M., Krey, V., Zilliacus, J.J. and Riahi, K., 2017. Understanding the origin of Paris Agreement emission uncertainties. Nature Communications, 8, p.15748.
https://www.nature.com/articles/ncomms15748

Reply:
@jkikstra the suggested publication is already part of the description.  :cite:`rogelj_indc_2017`. This describes the methodology.
The second publication is a reference we require for "which" policies we have implemented. This was supposed to be part of a paper published by a colleague at PBL. The paper is still not out, so I was wondering if there is another source we can quote.  @volker-krey Possibly we can link to http://www.climatepolicydatabase.org/